src/components/Card.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react'
+import CardModal from '../components/CardModal'
+import '../styles/Card.css'
+
+type CardProps = {
+  alt?: string
+  media_type?: string
+  title?: string
+  date?: string
+  src: string
+  highDefSrc?: string
+  explanation?: string
+  copyright: string
+  likedImage: boolean
+  innerRef?: (node: HTMLDivElement) => void
+  likeAction?: () => void
+}
+
+function Card({
+  alt,
+  media_type,
+  title,
+  date,
+  src,
+  highDefSrc,
+  explanation,
+  copyright,
+  likedImage,
+  innerRef,
+  likeAction,
+}: CardProps) {
+  const [modalStatus, setModalStatus] = useState(false)
+
+  const dayInLetters = date ? `${new Date(date).toDateString().slice(0, 3)},` : 'No Date'
+  const dayInNumbersAndYear = date ? new Date(date).toDateString().slice(7) : 'No Date'
+  let media = <img src={src} alt={alt} onClick={() => setModalStatus(true)}></img>
+
+  if (media_type === 'video')
+    media = <iframe src={src} title={title ? title : alt} onClick={() => setModalStatus(true)}></iframe>
+
+  return (
+    <>
+      <div ref={innerRef} className="card">
+        {media}
```

Comment:
Take advantage of JSX to control what elements render while keeping HTML huddled together.
```suggestion
        {media_type === 'video' ? (
          <iframe src={src} title={title ? title : alt} onClick={() => setModalStatus(true)}/>
        ) : (
          <img src={src} alt={alt} onClick={() => setModalStatus(true)}/>
        )}
```